joku/cogs/games/stocks.py

```diff
@@ -235,21 +235,21 @@ def _identify_stock(self, channels: typing.Sequence[discord.abc.GuildChannel], n
         for user in users:        
             stocks = await ctx.bot.database.get_user_stocks(user)
             # Append a column amount to list users
-            user + [sum(userstock.amount for userstock in stocks]
+            user = [user + [sum(userstock.amount for userstock in stocks]]
             # Append a column worth to list users
-            user + [sum(userstock.amount * userstock.stock.price for userstock in stocks)]
+            user = [user + [sum(userstock.amount * userstock.stock.price for userstock in stocks)]]
                 
-        if what == "amount" or "owned": #order by stock amount
+        if what == ("amount" or "owned"): #order by stock amount
```

Comment:
God damn I'm a fucking idiot... I've written the correct code so many fucking times, I have no idea how fucked that up so hard. bleh. 

if ((what == "amount") or (what == "owned")): 

I should quit programming and give up on life.